packages/garbo/src/tasks/freeFight.ts

```diff
@@ -801,6 +802,34 @@ const FreeFightTasks: GarboFreeFightTask[] = [
   // tied-up leaviathan (scaling, has 100 damage source cap and 2500 hp)
   // li'l ninja costume
   // closed-circuit pay phone (make into it's own Quest)
+  {
+    name: "CyberRealm Overclock Fights",
+    ready: () =>
+      have($item`server room key`) && // TODO support day pass? Not require tanktop
+      have($item`zero-trust tanktop`) &&
+      have($skill`Torso Awareness`) &&
+      have($skill`OVERCLOCK(10)`),
+    completed: () => get("_cyberFreeFights") >= 10,
+    do: $location`Cyberzone 1`, // TODO Support other zones with better equipment and valuing hacker drops
+    tentacle: false,
+    choices: () => ({ 1545: 1 }), // Take damage, get 0's
```

Comment:
```suggestion
    choices: { 1545: 1 }, // Take damage, get 0's
```
doesn't matter too much, but if nothing ever changes it's nicer to not make it a function imo